google/cloud/bigquery/client.py

```diff
@@ -2139,7 +2140,7 @@ def get_job(
         project: Optional[str] = None,
         location: Optional[str] = None,
         retry: retries.Retry = DEFAULT_RETRY,
-        timeout: TimeoutType = DEFAULT_TIMEOUT,
+        timeout: Optional[Union[TimeoutType, object]] = DEFAULT_GET_JOB_TIMEOUT,
```

Comment:
I think we discussed that `object` isn't necessary here, only in the job classes, right?

Reply:
Thanks! Cleaned up imports.

Reply:
Indeed, I have reverted this.